swiss_locator/core/settings.py

```diff
@@ -103,3 +103,5 @@ def __init__(self):
         self.add_setting(Stringlist("feature_search_layers_list", Scope.Global, None))
 
         self.add_setting(Bool("layers_include_opendataswiss", Scope.Global, True))
+
+        self.add_setting(Bool("with_qtwebkit", Scope.Global, True))
```

Comment:
do we really need a setting for this? what about moving this to a global property at plugin startup?

Reply:
Not really, please let me know if this is what you meant: https://github.com/opengisch/qgis-swiss-locator/pull/82/commits/36222e05c7f12a6c2fe3621d800f9ec80af263dd